meta-freescale-mbl/recipes-bsp/u-boot/u-boot-fslc_%.bbappend

```diff
@@ -1,15 +1,19 @@
-# Copyright (c) 2018 Arm Limited and Contributors. All rights reserved.
+# Copyright (c) 2018-2019 Arm Limited and Contributors. All rights reserved.
 #
 # SPDX-License-Identifier: MIT
 
 # MBL_UBOOT_VERSION should be updated to match version pointed to by SRCREV
 MBL_UBOOT_VERSION = "2018.11-rc1"
 
 SRCREV = "c0c4ee5fce01ec0818c4f27ce029d9b16c8849ad"
+SRCREV_imx7d-pico-mbl = "c72a23701bd9b15dab35e0d43c468ab051739af3"
 
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
 SRC_URI = "git://git.linaro.org/landing-teams/working/mbl/u-boot.git;protocol=https;nobranch=1 "
+SRC_URI_imx7d-pico-mbl = "git://git.denx.de/u-boot.git"
```

Comment:
should be gitlab.denx.de instead ? since it's now the official repo.

See https://lists.denx.de/pipermail/u-boot/2019-June/373065.html

Reply:
https://gitlab.denx.de/u-boot/u-boot.git

Reply:
Already revised.

Reply:
@niej ,

Are you going to change to gitlab.denx.de/u-boot/u-boot ?